src/Config/ConfigAdapter.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Config;
+
+/**
+ * Provides an adapter for the legacy Contao Config class so it can be injected
+ * and unit tested.
+ *
+ * @author Yanick Witschi <https://github.com/toflar>
+ */
+class ConfigAdapter
+{
+    /**
+     * @var \Config
+     */
+    private $config;
+
+    /**
+     * Save the local configuration file
+     */
+    public function save()
+    {
+        $this->config->save();
+    }
+
+    /**
+     * Return true if the installation is complete.
+     *
+     * @return boolean
+     */
+    public function isComplete()
+    {
+        return $this->config->isComplete();
+    }
+
+    /**
+     * Return all active modules as array
+     *
+     * @return array An array of active modules
+     *
+     * @deprecated Use ModuleLoader::getActive() instead
+     */
+    public function getActiveModules()
+    {
+        return $this->config->getActiveModules();
+    }
+
+    /**
+     * Add a configuration variable to the local configuration file
+     *
+     * @param string $strKey The full variable name
+     * @param mixed  $varValue The configuration value
+     */
+    public function add($strKey, $varValue)
+    {
+        $this->config->add($strKey, $varValue);
+    }
+
+    /**
+     * Alias for Config::add()
+     *
+     * @param string $strKey The full variable name
+     * @param mixed  $varValue The configuration value
+     */
+    public function update($strKey, $varValue)
+    {
+        $this->config->update($strKey, $varValue);
+    }
+
+    /**
+     * Remove a configuration variable
+     *
+     * @param string $strKey The full variable name
+     */
+    public function delete($strKey)
+    {
+        $this->config->delete($strKey);
+    }
+
+    /**
+     * Return a configuration value
+     *
+     * @param string $strKey The short key (e.g. "displayErrors")
+     *
+     * @return mixed|null The configuration value
+     */
+    public function get($strKey)
+    {
+        return \Config::get($strKey);
```

Comment:
Why are you not using `$this->config` here? Static methods can be called non-statically.